config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -825,7 +825,6 @@ postsubmits:
     decoration_config:
       timeout: 1h
     always_run: true
-    optional: true
```

Comment:
I see, the problem here is that is optional and always_run :smile:

Reply:
yes